src/modules/RfqModule.sol

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.13;
+
+// Libraries
+import "openzeppelin/utils/math/SafeCast.sol";
+import "lyra-utils/decimals/SignedDecimalMath.sol";
+import "lyra-utils/decimals/DecimalMath.sol";
+
+// Inherited
+import {BaseModule} from "./BaseModule.sol";
+import {IRfqModule} from "../interfaces/IRfqModule.sol";
+
+// Interfaces
+import {IBaseManager} from "v2-core/src/interfaces/IBaseManager.sol";
+import {IDataReceiver} from "v2-core/src/interfaces/IDataReceiver.sol";
+import {ISubAccounts} from "v2-core/src/interfaces/ISubAccounts.sol";
+import {IAsset} from "v2-core/src/interfaces/IAsset.sol";
+import {IPerpAsset} from "v2-core/src/interfaces/IPerpAsset.sol";
+import {IMatching} from "../interfaces/IMatching.sol";
+
+import "forge-std/console2.sol";
+
+/**
+ * @title RfqModule
+ * @dev Allows a user to request a bundle of trades to all be executed atomically
```

Comment:
I think it's better to update comments here to say that "makers" always being the ones "buying" perp or option assets with quote. It would be nice if this is clarified first, instead of assuming it can handle any kind of "trades"

Reply:
They're not necessarily buying, as they can bid a negative amount to get paid to take on a negative position